lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -185,6 +185,12 @@ export abstract class ClientApplication {
                 this.logger.verbose(e);
             }
         }
+
+        if(!this.config.cache.claimsBasedCachingEnabled) {
+            this.logger.verbose("Claims-based caching is disabled. Clearing the previous cache with claims");
+            await this.browserStorage.clearTokensAndKeysWithClaims();
```

Comment:
Can we add some telemetry submeasurements around this?

Reply:
I almost think a message of `Warning` is merited if any keys are actually cleared. This will help highlight any issues in telemetry, particularly if this is occurring more than we expect. (For our use case I only ever expect to see this once per user and would like to verify that if possible.)

Reply:
Sure. Can change this to warning and will add telemetry as needed.

Reply:
Just to clarify I meant that it would be `Warning` only if tokens and keys were cleared (if easily possible) rather than a `Warning` as it is right now (which would occur on every app load.) Thanks!

Reply:
Addressed both: Added telemetry and a warning.